crates/cli/src/cli_args/add.rs

```diff
@@ -0,0 +1,180 @@
+use crate::State;
+use clap::Args;
+use miette::Context;
+use pacquet_package_json::DependencyGroup;
+use pacquet_package_manager::Add;
+use std::path::PathBuf;
+
+#[derive(Debug, Args)]
+pub struct AddDependencyOptions {
+    /// Install the specified packages as regular dependencies.
+    #[clap(short = 'P', long)]
+    save_prod: bool,
+    /// Install the specified packages as devDependencies.
+    #[clap(short = 'D', long)]
+    save_dev: bool,
+    /// Install the specified packages as optionalDependencies.
+    #[clap(short = 'O', long)]
+    save_optional: bool,
+    /// Using --save-peer will add one or more packages to peerDependencies and install them as dev dependencies
+    #[clap(long)]
+    save_peer: bool,
+}
+
+impl AddDependencyOptions {
+    /// Convert the `--save-*` flags to an iterator of [`DependencyGroup`]
+    /// which selects which target group to save to.
+    fn dependency_groups(&self) -> impl Iterator<Item = DependencyGroup> {
+        let &AddDependencyOptions { save_prod, save_dev, save_optional, save_peer } = self;
+        let has_prod = save_prod || (!save_dev && !save_optional && !save_peer); // no --save-* flags implies --save-prod
+        let has_dev = save_dev || (!save_prod && !save_optional && save_peer); // --save-peer with nothing else implies --save-dev
+        let has_optional = save_optional;
+        let has_peer = save_peer;
+        std::iter::empty()
+            .chain(has_prod.then_some(DependencyGroup::Default))
```

Comment:
Not sure Default is a good name for this enum. I would rename it to prod
```suggestion
            .chain(has_prod.then_some(DependencyGroup::Prod))
```

Reply:
I wondered the same too. "Why did @anonrig name it `Default` instead of `Prod`?". I guess it has to with the fact that `"dependencies"` wasn't called `"prodDependencies"`? Anyway, I think the name change is a no-brainer, this is a good time to rename it now.

Reply:
renamed by https://github.com/pnpm/pacquet/pull/143/commits/886f1d4a88c07fdf1ead393911ac7a10466cb3c1

Reply:
I mainly used default because there isn't any prodDependencies, since there isn't any prod dependencies. there is dependencies to correlate with the existing package.json naming. 

I'm OK with renaming it tho.